lib/src/utilities/dart_type_utilities.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library linter.src.utilities.dart_type_utilities;
+
+import 'package:analyzer/dart/element/type.dart';
+
+class DartTypeUtilities {
+  static List<InterfaceType> findImplementedInterfaces(InterfaceType type,
```

Comment:
Probably not. It was written to figure out which members are inherited more than which interfaces were implemented. It certainly has to compute the latter in order to figure out the former, though, so it potentially could be made to provide that information. On the other hand, it seems a bit heavy weight if all you want is the latter, so perhaps a method on InterfaceType (or even DartType).

But it does seem strange to add utility methods for a framework we control rather than just adding the method to an appropriate class.